packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -440,13 +440,17 @@ export const transformers = (model: ModelData): string[] => {
 	}
 
 	if (model.pipeline_tag && LIBRARY_TASK_MAPPING.transformers?.includes(model.pipeline_tag)) {
-		const pipelineSnippet = [
-			"# Use a pipeline as a high-level helper",
-			"from transformers import pipeline",
-			"",
-			`pipe = pipeline("${model.pipeline_tag}", model="${model.id}"` + remote_code_snippet + ")",
-		].join("\n");
-		return [pipelineSnippet, autoSnippet];
+		const pipelineSnippet = ["# Use a pipeline as a high-level helper", "from transformers import pipeline", ""];
+
+		if (model.tags.includes("conversational") && model.config?.tokenizer_config?.chat_template) {
```

Comment:
To be confirmed but is `model.config?.tokenizer_config?.chat_template` really needed here? The logic in moon-landing to determine if we should display the conversational widget is 

```js
model.pipeline_tag === "text-generation" && model.tags.includes("conversational")
```

(see [here](https://github.com/huggingface/moon-landing/blob/e01f92263a8c414121873e2c83e5c99bff057d67/packages/widgets/src/lib/components/InferenceWidget/InferenceWidget.svelte#L85))

Is it possible that for some models the chat_template is hardcoded or use a default one and in this case they won't have it in tokenizer_config?

Reply:
Default templates were just deprecated, see the convo above https://github.com/huggingface/huggingface.js/pull/728#discussion_r1621204099

Reply:
Oups sorry, should have checked the resolved comments first :smile: Looks good then :)